modules/web/js/ballerina/diagram2/views/default/components/decorators/annotation-attribute-key.jsx

```diff
@@ -176,7 +196,7 @@ class AnnotationAttributeKey extends React.Component {
                 placeholder={'Key'}
                 initialValue=''
                 onSuggestionSelected={(event, { suggestionValue }) => { this.onKeySelected(suggestionValue); }}
-                onBlur={() => this.onKeyBlur()}
+                onBlur={e => this.onKeyBlur(e)}
```

Comment:
don't we need to bind in the constructor ?

Reply:
Checked the revision its bind in constructor. 
https://github.com/hemikak/composer/blob/58244e2ac4218d7ca9e087fe82e76603b338ca19/modules/web/js/ballerina/diagram2/views/default/components/decorators/annotation-attribute-key.jsx#L52